src/plone/meta/config_package.py

```diff
@@ -531,7 +531,9 @@ def handle_gh_actions(self):
         combinations = []
         for plone_version, python_versions in test_matrix.items():
             no_dot_plone = plone_version.replace(".", "")
-            for py_version in {python_versions[0], python_versions[-1]}:
+            top_bottom_versions = {python_versions[0], python_versions[-1]}
+            sorted_versions = sorted(list(top_bottom_versions), reverse=True)
```

Comment:
You don't need list() here. sorted() accepts any iterable and returns a list.

```suggestion
            sorted_versions = sorted(top_bottom_versions, reverse=True)
```